rusk/src/bin/args/command.rs

```diff
@@ -18,6 +18,10 @@ pub enum Command {
         /// Keeps untracked keys
         #[clap(short, long, value_parser = BoolishValueParser::new(), env = "RUSK_KEEP_KEYS")]
         keep: bool,
+
+        /// URL of the server to download the CRS from
+        #[clap(long, value_parser = BoolishValueParser::new(), env = "RUSK_CRS_URL")]
```

Comment:
This is not a boolish parser

It's a normal string, and you probably should put a default value in order to not require this arg (something like "https://nodes.dusk.network/static/network.crs")

Reply:
mmm, maybe would be better to compute a fresh CRS if the URL is not provided? I'm saying it because, what happens if the server is down? We won't be able to run the tests.

Nevertheless, this is not the best approach if we want to enforce the secure way of running the node. Maybe awe could add the trusted setup into the repository? It's just 6MB...